adopt-a-dog/blog.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="zxx">
+<head>
+  <meta charset="utf-8">
+  <title>Adaptadog</title>
+  <!-- <link href="https://fonts.googleapis.com/css?family=Lato" rel="stylesheet"> -->
+  <!-- <link rel="stylesheet" type="text/css" href="styles.css"> -->
+</head>
+<body>
+    <!-- <header> -->
+    <div class="container">
+      <img src="images/christopher-harris-57366-unsplash.jpg" alt="header image" height="700">
```

Comment:
You can go ahead and ignore this image for now. We haven't covered how to do this yet. But don't forget the dog logo...